src/main/java/com/direwolf20/buildinggadgets/common/tools/ToolRenders.java

```diff
@@ -98,15 +100,24 @@ public static void renderBuilderOverlay(RenderWorldLastEvent evt, EntityPlayer p
             dispatcher.renderBlockBrightness(Blocks.YELLOW_STAINED_GLASS.getDefaultState(), 1f);
             GlStateManager.popMatrix();
         }
-
-        RayTraceResult lookingAt = VectorUtil.getLookingAt(player);
+        LiquidInteractions.GadgetBuilding flIntState = GadgetBuilding.getFluidPlacementMode(heldItem);
+        RayTraceResult lookingAt;
+        if (flIntState == LiquidInteractions.GadgetBuilding.IGNORE) {
```

Comment:
This has multiple instances of code dupe meaning we can create a method to contain the code to clean this up. Add it as a static to GenericGadget.

Reply:
Better... A Method on the enum...

Reply:
Could not find the line in final version...
src/main/java/com/direwolf20/buildinggadgets/common/tools/ToolRenders.java:102

Reply:
?